test/runtests.jl

```diff
@@ -1,7 +1,9 @@
-using Test, RangeEnclosures
-using RangeEnclosures: relative_precision, available_solvers
+using IntervalOptimisation, Test, RangeEnclosures
+using RangeEnclosures: relative_precision
 
 using DynamicPolynomials: @polyvar
 
+available_solvers = (:IntervalArithmetic, :TaylorModels, :BranchandBound, :IntervalOptimisation)
```

Comment:
Why not use `available_solvers` like before?

Reply:
@mforets do you remember?

Reply:
Ideally we shouldn't need such tuple at all, and just rely on dispatch?

Let's focus on getting the compilation issue right on this PR.

Reply:
I mean, even if you use dispatch, you probably still want a tuple `solvers_to_test = (Solver1(), Solver2(), Solver3())` to test the `enclose` function? Note this is in the `runtest.jl` file. I agree that such a tuple is not needed inside the source code (indeed atm as far as I can say even now it's not used), but this can be delegated to future RP

Reply:
> I mean, even if you use dispatch, you probably still want a tuple solvers_to_test = (Solver1(), Solver2(), Solver3()) to test the enclose function? 

Correct; the idea here was to iterate over some specified set of solver backends for the same problem.